docs/readme.md

```diff
@@ -0,0 +1,83 @@
+# Widgets
+
+## base nodes
+* [Node](/docs/widgets/baseNodes/node.md) (abstract)
```

Comment:
it looks like github actually resolves these urls correctly. A few of the links are camel case where the file is all lower and that breaks it but it is navigable as is.